### PR TITLE
DW-973 fix(Promotions): get default image from be

### PR DESCRIPTION
**Pull request by fcoronelmakingsense**
- Get the default image setting in be if has one, is not continue getting from local.

image

### CODE REVIEW

**Comment by cbernat** [comment]
How about some tests please? 😄

**Comment by cbernat** [comment]
https://cdn.fromdoppler.com/doppler-webapp/build3221/#/signup 
This is looking nice!

File: src/components/shared/Promotions/Promotions.js [hunk 1]
```diff
@@ -30,18 +30,14 @@ const Promotions = function ({ type, page, dependencies: { dopplerSitesClient }
     const fetchData = async () => {
       setState({ loading: true });
       const bannerData = await dopplerSitesClient.getBannerData(intl.locale, type, page || '');
-      if (!bannerData.success) {
+      if (!bannerData || !bannerData.success) {
```
Comment (jcamposmk) [inline src/components/shared/Promotions/Promotions.js:33]: Optional chaining operator, maybe: bannerData?.success, it's more simple ...

**Comment by fcoronelmakingsense** [comment]
> How about some tests please? 😄

there are some

**Comment by cbernat** [comment]
🚀

**Comment by cbernat** [comment]
As we discussed by chat, you could try add a test that validates if the page parameter does not return data, that you go and try searching for the default data.

**Comment by fcoronelmakingsense** [comment]
added test

**Pull request merged**